nuxeo-services/nuxeo-platform-usermanager-core/src/main/resources/OSGI-INF/usermanager-properties.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.ecm.platform.usermanager.properties">
+
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <documentation>
+      Controls whether we check the validity of password when changing it.
+
+      @since 8.4
+    </documentation>
+    <property name="nuxeo.usermanager.check.password">true</property>
+  </extension>
+
+</component>
```

Comment:
This file shouldn't be included in the backport as the ConfigurationService does not exist.